### DISCUSSION TITLE
Removal of team discussions - February 8, 2023

### DISCUSSION THREAD

**Discussion by shiva-sandupatla** [community/community#46970] [Discussions] [answered]
### Select Topic Area

Product Feedback

### Body

Recently, @github has announced [Sunset Notice – Team Discussions](https://github.blog/changelog/2023-02-08-sunset-notice-team-discussions) 

Team discussions are a great way to discuss privately within a team rather than exposing it to other organization members.

Due to **sunset notice** after May 8, 2023, it is impossible to create discussions within the team privately.

Is there any way to create discussions privately or add a feature to create a **"private category"** to create discussions privately for a team at the organization level discussions?

Hoping for a positive response.

Regards, @shiva-sandupatla

**Comment by objecttothis** [comment]
I just learned about this as well.  The whole benefit of team discussions was that it was private.  @github-staff please add in the ability to control who can see and contribute to a discussion. I'm thinking selections like org, everyone (if public repo), or a selected team.

**Reply by shiva-sandupatla** [reply]
@objecttothis, thanks for the reply

**Comment by LindseyB** [comment, marked as answer]
Right now the best option for a private discussion would be creating a private repository and enabling discussions in it and only granting access to those users that you want to have access.

**Reply by objecttothis** [reply]
Is this intended to be the solution or a workaround for which github is going to restore the ability to make a team discussion private?

**Reply by LindseyB** [reply]
It's the solution for the foreseeable future.
